apps/mull-ui/src/app/components/contact-row/contact-row.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../variables.scss';
+
+.contact-container {
+  display: flex;
+  margin-top: 2rem;
+  justify-content: space-between;
+  .user-information {
+    display: flex;
+    flex-direction: row;
+  }
+  .user-profile-picture {
+    width: 3.5rem;
+    height: 3.5rem;
+    object-fit: cover;
+    border-radius: 50%;
+    border: 1px solid black;
+  }
+  .user-details {
+    display: flex;
+    padding: 0.3rem;
+    flex-direction: column;
+    .userName {
+      margin-top: 0.5rem;
+      font-weight: normal;
+      font-size: 18px;
```

Comment:
💡 Probably better to use rem instead of px, this suggestion applies to the whole file